tests/test_main.py

```diff
@@ -0,0 +1,36 @@
+"""Tests for the main file of the project."""
+from typer.testing import CliRunner
+
+from gatorgrade import main
+
+runner = CliRunner()
+
+
+def test_gatorgrade_runs():
+    """Test that ensures that the default command runs correctly."""
+    result = runner.invoke(main.app)
+
+    assert result.exit_code == 0
+
+
+def test_generate_creates_valid_yml():
+    """Test that ensures that the generate command creates
+    the .yml file correctly."""
+    result = runner.invoke(main.app, "--force")
+
+    assert result.exit_code == 2
+
+
+def test_generate_fails_with_existing_yml():
+    """Test that ensures that a second yml file isn't generated
+    without the force command."""
+    result = runner.invoke(main.app)
+
+    assert result.exit_code == 0
+
+
+def test_generate_force_option_creates_yml():
+    """Test that ensures the force command works correctly."""
+    result = runner.invoke(main.app, "--force")
+
+    assert result.exit_code == 2
```

Comment:
The expected exit code should probably be 0, unless you are expecting something to go wrong when using the `--force` option.